src/Native/Common/utilcode.h

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+#include "daccess.h"
+
+#define THUMB_CODE 1
```

Comment:
`src\Native\Runtime\MiscHelpers.cpp` has local definition of this as `THUMB_BIT`. It would be nice to replace it with this global one.

Reply:
Done